packages/client/src/views/SysAdmin/Performance/Vsexports/VSExport.tsx

```diff
@@ -0,0 +1,180 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import React from 'react'
+import { SysAdminContentWrapper } from '@client/views/SysAdmin/SysAdminContentWrapper'
+import { FormTabs } from '@opencrvs/components/lib/forms'
+import styled from 'styled-components'
+import { useIntl } from 'react-intl'
+import { BodyContent } from '@opencrvs/components/lib/layout'
+import { Content } from '@opencrvs/components/lib/interface/Content'
+import { messages } from '@client/i18n/messages/views/config'
+import {
+  ListViewItemSimplified,
+  ListViewSimplified
+} from '@opencrvs/components/lib/interface/ListViewSimplified/ListViewSimplified'
+import { Query } from '@client/components/Query'
+import { GET_TOTAL_VSEXPORT } from './queries'
+import {
+  Label,
+  Value
+} from '@client/views/SysAdmin/Config/Application/Components'
+import { LoadingIndicator } from '@client/views/OfficeHome/LoadingIndicator'
+
+import { MINIO_URL } from '@client/utils/constants'
+import { DynamicHeightLinkButton } from '@client/views/Settings/items/components'
+import { ErrorToastNotification } from '@opencrvs/components/lib/interface'
+
+import { buttonMessages, errorMessages } from '@client/i18n/messages'
+import { Event, GetVsExportsQuery, VsExport } from '@client/utils/gateway'
+
+const UserTable = styled(BodyContent)`
```

Comment:
@CornelMaze 
Why is this needed? Is it because we arn't using < Frame / >

Reply:
I'll check it out.

Reply:
@jpye-finch Yes. Is the Frame available?

Reply:
yes
https://ui-kit.farajaland-qa.opencrvs.org/?path=/docs/layout-frame--page-template-content-large